addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -32,6 +33,15 @@ class DiscussChannelMember(models.Model):
     is_self = fields.Boolean(compute="_compute_is_self", search="_search_is_self")
     # channel
     channel_id = fields.Many2one("discuss.channel", "Channel", ondelete="cascade", required=True, bypass_search_access=True)
+    channel_role = fields.Selection(
+        [("owner", "Owner"), ("admin", "Admin")],
+        default=None,
+        string="Channel Role",
+        help="The role of the member in the channel."
+        "The owner can add and remove admins."
+        "The owner and admins can edit the channel and its members.",
```

Comment:
Single line for help might be ok now. Doesn't this miss spaces after each phrase in the final result?